package.json

```diff
@@ -10,10 +10,12 @@
   "author": "",
   "license": "BSD",
   "dependencies": {
-    "osm-stream": "0.0.10dev0",
+    "osm-stream": "~0.0.12",
     "ration": "0.0.0",
     "underscore": "~1.4.4",
     "reqwest": "~0.7.2",
-    "moment": "~2.0.0"
+    "moment": "~2.0.0",
+    "lru-cache": "~4.0.0",
+    "query-string": "~3.0.0"
```

Comment:
you should be able to `require('querystring')` and browserify itself will include the builtin